app/controllers/CheckYourAnswersController.scala

```diff
@@ -103,6 +117,47 @@ class CheckYourAnswersController @Inject()(
         ).flatten
       )
 
-      Ok(view(vatRegistrationDetailsList, list))
+      val isValid = validate()
+      Ok(view(vatRegistrationDetailsList, list, isValid))
+  }
+
+  def onSubmit(incompletePrompt: Boolean): Action[AnyContent] = cc.authAndGetData().async {
+    implicit request =>
+      registrationService.fromUserAnswers(request.userAnswers, request.vrn) match {
+        case Valid(registration) =>
+          registrationConnector.submitRegistration(registration).flatMap {
+            case Right(_) =>
+              auditService.audit(RegistrationAuditModel.build(RegistrationAuditType.CreateRegistration, registration, SubmissionResult.Success, request))
+              Future.successful(Ok) // TODO: replace with sendEmailConfirmation
```

Comment:
Replace with `sendEmailConfirmation` once https://github.com/hmrc/ioss-registration-frontend/pull/22 is merged